packages/web-components/fast-foundation/src/data-grid/data-grid.spec.md

```diff
@@ -121,6 +211,21 @@ Can be either "none", "default" or "sticky" (the `GeneratHeaderOptions` enum). A
 - `grid-template-columns`  
 String that gets applied to the the css gridTemplateColumns attribute of child rows. Corresponds to the [grid-template-columns css attribute](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns)
 
+- `selection-mode`
+Sets how the grid handles selection.
+"none" - The default.  The grid does not enable selection.
+"single-row" - Single row elements are selectable.
+"multi-row" - Multiple rows can be selected.
+
+- `click-select`
+Whether the grid selects items based on click events or not.  Authors may wish to invoke selection directly from their own ui, like having a specific checkbox to select a row, in which case `click-select` would be disabled and `toggleSelected()` called on the desired row/cell.  Default is `true`.
+
+- `select-row-header`
```

Comment:
I don't see this in code. If I missed it or if we add it I suggest "select*able*-row-header" or "row-header-select*able*"

Reply:
Ended up pulling that because it will only become relevant when we do cell selection.